src/Qoollo.PerformanceCounters/CompositeCounters/Counters/CompositeDeltaCounter.cs

```diff
@@ -0,0 +1,152 @@
+            : base(name, description)
+        {
+            if (counters == null)
+                throw new ArgumentNullException("counters");
+
+            _wrappedCounters = counters.ToArray();
+
+            if (_wrappedCounters.Length == 0)
+                _wrappedCounters = new DeltaCounter[] { new NullCounters.Counters.NullDeltaCounter(name, description),  };
+        }
+
+        /// <summary>
+        /// Уменьшить значение на 1
+        /// </summary>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long Decrement()
+        {
+            long result = _wrappedCounters[0].Decrement();
+            for (int i = 1; i < _wrappedCounters.Length; i++)
+                _wrappedCounters[i].Decrement();
+            return result;
+        }
+
+        /// <summary>
+        /// Уменьшить значение на "value"
+        /// </summary>
+        /// <param name="value">Значение, на которое уменьшаем</param>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long DecrementBy(long value)
+        {
+            long result = _wrappedCounters[0].DecrementBy(value);
+            for (int i = 1; i < _wrappedCounters.Length; i++)
+                _wrappedCounters[i].DecrementBy(value);
+            return result;
+        }
+
+        /// <summary>
+        /// Увеличить значение на 1
+        /// </summary>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long Increment()
+        {
+            long result = _wrappedCounters[0].Increment();
+            for (int i = 1; i < _wrappedCounters.Length; i++)
+                _wrappedCounters[i].Increment();
+            return result;
+        }
+
+        /// <summary>
+        /// Увеличить значение на "value"
+        /// </summary>
+        /// <param name="value">Значение, на которое увеличиваем</param>
+        /// <returns>Текущее значение или Counter.FailureNum</returns>
+        public override long IncrementBy(long value)
+        {
+            long result = _wrappedCounters[0].IncrementBy(value);
+            for (int i = 1; i < _wrappedCounters.Length; i++)
+                _wrappedCounters[i].IncrementBy(value);
+            return result;
+        }
+
+        /// <summary>
+        /// Задать значение
+        /// </summary>
+        /// <param name="value">Новое значение</param>
+        public override void SetValue(long value)
+        {
+            for (int i = 0; i < _wrappedCounters.Length; i++)
+                _wrappedCounters[i].SetValue(value);
+        }
+
+        /// <summary>
+        /// Получает текущее значение счетчика производительности
+        /// </summary>
+        public override long CurrentValue
+        {
+            get { return _wrappedCounters[0].CurrentValue; }
+        }
+
+        public override long Measure()

```

Comment:
Please, add the xml-comment to the method (the same as in the base class) to prevent the warning during compilation

Reply:
Ok, added.